sparkle/src/components/AssistantPreview2.tsx

```diff
@@ -27,6 +27,7 @@ type ListVariantAssistantPreviewProps = BaseAssistantPreviewProps & {
 type GalleryVariantAssistantPreviewProps = BaseAssistantPreviewProps & {
   variant: "gallery";
   renderActions?: (isHovered: boolean) => React.ReactNode;
+  onPlayClick?: (e: SyntheticEvent) => void;
```

Comment:
why pass a syntheticevent?

Reply:
To stop propagating the event since we have a CTA on the parent div.